lua/orgmode/org/hyperlinks/init.lua

```diff
@@ -216,9 +216,10 @@ end
 function Hyperlinks.insert_link(link_location)
   local selected_link = Link:new(link_location)
   local desc = selected_link.url:get_target_value()
+
   if selected_link.url:is_id() then
     local id_link = ('id:%s'):format(selected_link.url:get_id())
-    desc = link_location:gsub('^' .. vim.pesc(id_link) .. '%s+', '')
+    desc = selected_link.url:get_target_value()
```

Comment:
And now as I review it myself, I see, that this is redundant ... I will remove it. :sweat_smile: